<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CJCE1FP" failures="1" tests="1" name="Test2" time="28.759" errors="0" timestamp="2024-05-28T12:12:35 IST">
  <testcase classname="uk.co.automationtesting.ShopLoginTest" name="addRemoveItem" time="23.754">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.fail(Assert.java:95)
at uk.co.automationtesting.ShopLoginTest.addRemoveItem(ShopLoginTest.java:119)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- addRemoveItem -->
</testsuite> <!-- Test2 -->
